### PR TITLE
Make "Roslyn" default setup project

### PR DESCRIPTION
**Pull request by davkean**
By default, VS sets the first project listed in the solution as the default project. This saves new contributors from having to manually set the default project.

### CODE REVIEW

File: Roslyn.sln [hunk 1]
```diff
@@ -958,10 +958,6 @@ Global
 		{49E7C367-181B-499C-AC2E-8E17C81418D6}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{49E7C367-181B-499C-AC2E-8E17C81418D6}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{49E7C367-181B-499C-AC2E-8E17C81418D6}.Release|Any CPU.Build.0 = Release|Any CPU
-		{96EB2D3B-F694-48C6-A284-67382841E086}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (davkean) [inline Roslyn.sln]: This project was clearly removed in previous changes without updating the solution.

**Comment by davkean** [comment]
Why is Roslyn in the root if it isn't the default? And why would it be VisualStudioSetup?

**Comment by sharwell** [comment]
VisualStudioSetup is the project you can <kbd>F5</kbd> to debug Roslyn.

**Comment by davkean** [comment]
Roslyn includes CompilerExtension, Debugging, Interactive and IDE. VisualStudioSetup is only IDE.

**Comment by jmarolf** [comment]
@sharwell this looks like an oversight, when I added `Roslyn.csproj` the intent was that it would be the default startup project that depends on all other VSIX projects so compiler changes could also be picked up on F5.  Has anything changed here?

**Comment by sharwell** [comment]
@jasonmalinowski ?

**Comment by jasonmalinowski** [comment]
@sharwell This seems good.

**Comment by davkean** [comment]
@jinujoseph Can I get a sign-off on this for 15.8? This makes changes to the solution which is a pain to merge so it would be better in "current" release vs a future release.

**Comment by jinujoseph** [comment]
Approved to merge for 15.8.Preview4

**Pull request merged**